proposed/phpdoc-tags.md

```diff
@@ -824,15 +824,15 @@ function count(array $items)
 ```
 
 The following example demonstrates the use of an "Inline PHPDoc" to document
-an option array with two elements: 'required' and 'label'.
+an option array with two elements, representing 'required' and 'label'.
 
 ```php
 /**
  * Initializes this class with the given options.
  *
  * @param array $options {
- *     @var bool   $required Whether this element is required
- *     @var string $label    The display name for this element
+ *     @var bool   Whether this element is required
```

Comment:
This part is useless - what is bool and what is string in this array? Key/Value? some particular named keys?

Reply:
I think that represents a problem with the Inline PHPDoc syntax overall, so we should probably start that conversation anew separately.  In here, my main thing was recognizing that the `@var` syntax was showing variable names that actually do not exist in the code... they were simply given names to hint at their intent, while ordering of the tags is what's supposed to align with ordering of the array members.

My inclination is to consider dropping the Inline PHPDocs, and either let them go or start the discussion anew... like we're doing with generics.